taxi_service/settings.py

```diff
@@ -45,8 +45,11 @@
     "debug_toolbar",
     "crispy_forms",
     "taxi",
+    "crispy_bootstrap4",
 ]
 
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The `CRISPY_TEMPLATE_PACK` setting is defined twice. You can remove one of the definitions to avoid redundancy.